client/my-sites/earn/home.tsx

```diff
@@ -0,0 +1,252 @@
+			  )
+			: translate(
+					'Charge for services, collect membership dues, or take recurring donations. Automate recurring payments, and use your site to earn reliable revenue. {{em}}Available to any site with a paid plan{{/em}}.',
+					{
+						components: {
+							em: <em />,
+						},
+					}
+			  );
+		const learnMoreLink = isFreePlan
+			? 'https://en.support.wordpress.com/recurring-payments/'
+			: null;
+		return {
+			title,
+			body,
+			image: {
+				path: '/calypso/images/earn/recurring.svg',
+			},
+			cta,
+			learnMoreLink,
+		};
+	};
+
+	/**
+	 * Return the content to display in the Store card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getStoreCard = () => {
+		if ( isAJetpackSite ) {
+			return null;
+		}
+
+		const cta = hasUploadPlugins
+			? {
+					text: translate( 'Set Up a Simple Store' ),
+					action: () => page( `/store/${ selectedSiteSlug }` ),
+			  }
+			: {
+					text: translate( 'Upgrade to Business Plan' ),
+					action: () => page( `/checkout/${ selectedSiteSlug }/business/` ),
+			  };
+		return {
+			title: translate( 'Sell a few items' ),
+			body: translate(
+				'Create a basic store with minimal setup -- add your products, and use the built-in shipping, coupon, and tax tools. {{em}}Available to sites with a Business plan{{/em}}.',
+				{
+					components: {
+						em: <em />,
+					},
+				}
+			),
+			image: {
+				path: '/calypso/images/earn/woo.svg',
+			},
+			cta,
+			learnMoreLink: 'https://en.support.wordpress.com/store/',
+		};
+	};
+
+	/**
+	 * Return the content to display in the Ads card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getAdsCard = () => {
+		const cta = hasWordAds
+			? {
+					text: hasSetupAds ? translate( 'View Ad Dashboard' ) : translate( 'Earn Ad Revenue' ),
+					action: () => page( `/earn/ads-earnings/${ selectedSiteSlug }` ),
+			  }
+			: {
+					text: translate( 'Upgrade to Premium Plan' ),
+					action: () => page( `/checkout/${ selectedSiteSlug }/premium/` ),
+			  };
+		const title = hasSetupAds ? translate( 'View Ad Dashboard' ) : translate( 'Earn ad revenue' );
+		const body = hasSetupAds
+			? translate(
+					"Check out your ad earnings history, including total earnings, total paid to date, and the amount that you've still yet to be paid."
+			  )
+			: translate(
+					'Publish as you normally would, display advertisements on all your posts and pages, and make money each time someone visits your site. {{em}}Available to sites with a Premium plan{{/em}}.',
+					{
+						components: {
+							em: <em />,
+						},
+					}
+			  );
+		const learnMoreLink = ! hasWordAds ? 'https://wordads.co/' : null;
+		return {
+			title,
+			body,
+			image: {
+				path: '/calypso/images/earn/ads.svg',
+			},
+			cta,
+			learnMoreLink,
+		};
+	};
+
+	const promos: PromoSectionProps = {
+		header: {
+			title: translate( 'Start earning money' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Start earning money now' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).